conjure-codegen/src/errors.rs

```diff
@@ -18,12 +18,18 @@ use crate::context::Context;
 use crate::objects;
 use crate::types::{ErrorDefinition, ObjectDefinition};
 
+impl From<ErrorDefinition> for ObjectDefinition {
+    fn from(error: ErrorDefinition) -> Self {
+        ObjectDefinition::builder()
+            .type_name(error.error_name().clone())
+            .fields(error.safe_args().iter().chain(error.unsafe_args()).cloned())
+            .docs(error.docs().cloned())
+            .build()
+    }
+}
```

Comment:
could be a regular helper fn if implementing `from` seems incorrect, I just wanted to ensure that the generated object definition was consistent between the code below and during context creation

Reply:
Yeah I think I'd just make this a function personally.